ci-operator/config/openshift/external-secrets-operator/openshift-external-secrets-operator-main.yaml

```diff
@@ -92,7 +92,12 @@ tests:
     - as: install
       cli: latest
       commands: |
-        oc -n external-secrets-operator rollout status deployment external-secrets-operator-controller-manager
+        oc create namespace external-secrets-operator
+        oc label --overwrite ns external-secrets-operator pod-security.kubernetes.io/enforce=privileged
+        make operator-sdk
+        ./_cache/operator-sdk version
+        ./_cache/operator-sdk run bundle -n external-secrets-operator "$OO_BUNDLE" # install my-operator
```

Comment:
This doesn't seem to make sense to me, I can only see one repo using `./_cache/operator-sdk`: https://github.com/openshift/release/commit/222bd7e767335357148ed4a9a9a97bd88d72f510, but their motivation is to use the latest operator-sdk binary